---
title: "Coding in R for Microbiome"
output: html_document
---







R is a powerful tool for analyzing data. In microbiome research, complexity of data can be overwhelming with 1000s of taxa and 100s-10000s of samples and several associated metadata. Several R packages provide interface for handling of microbiome data [Shetty and Lahti 2019](https://www.ias.ac.in/public/Volumes/jbsc/044/05/0115.pdf). There is a non-exhaustive list of tools that I have complied together with input from microbiome community which can be found [here](https://microsud.github.io/Tools-Microbiome-Analysis/).   

## Common tasks at hand for analysis of data include:  

* __Data cleaning__  
  Example: Taxonomic labels like "g__" , "<empty>_" and several uncommon labels.  
* __Data organization__  
  Organizing data in a way that available tools can be used. For instance, converting `phyloseq` object into common *data.frame/tibble* format [either long or wide](https://tidyr.tidyverse.org/) so that tidyverse becomes accessible.  
* __Data handling and processing__  
  Data can be further improved by adding and/or organizing variables, changing variable names,
  transforming variables, etc.  
* __Statistics__  
  Checking if your observations are common or just by chance. While this aspect requires training in statistics, coding can be used for commonly statistical tests. Personally I think it is better to include a statistician in your project or a bioinformatician, because these are skills gained by experience and training.     
* __Automating steps__  
  One of the advantages of using coding skills, so that you can drink your beverage of choice while your analysis is being run by a machine.  
* __Visualization__  
  Conveying your message with the appropriate visualization of data.  
  
In the accompanying articles on this website, I will aim for introducing these aspects with focus on microbiome researchers using microbiome data (mostly 16S rRNA gene sequencing data).  

## Go to places  
I would highly recommend two more websites/links:   

* The [riffomonas](http://www.riffomonas.org/minimalR/) project by Dr. Pat Schloss.    
* The [Happy Belly Bioinformatics](https://astrobiomike.github.io/) project by Dr. Mike Lee.  
* The [microbiome tutorial page](https://microbiome.github.io/tutorials/) project by Dr. leo Lahti.   

These are comprehensive resources for microbiome data analysis.  

I will try to avoid repetition to the best of my abilities and will focus on harnessing the phyloseq, microbiome and microbiomeutilities R packages. 
  
  
  
  
  

