<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Basic tasks in R</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>

<link rel="stylesheet" href="styles.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Sudarshan A. Shetty</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="Publications.html">Publications</a>
</li>
<li>
  <a href="Research_Projects.html">Research</a>
</li>
<li>
  <a href="teaching.html">Teaching</a>
</li>
<li>
  <a href="Science_communication.html">Science communication</a>
</li>
<li>
  <a href="Other_Informations.html">Other Information</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Software and codes
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="Software.html">Software</a>
    </li>
    <li>
      <a href="coding_main.html">Coding in R for Microbiome</a>
    </li>
    <li>
      <a href="code_basics.html">Basic tasks in R</a>
    </li>
    <li>
      <a href="code_loops.html">For loops in R</a>
    </li>
    <li>
      <a href="code_customize_ordination.html">Customize ordination plots</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Basic tasks in R</h1>

</div>


<p>Once you have sequenced 16S rRNA gene libraries and processed the raw sequences with your choice of tool, <a href="https://msystems.asm.org/content/2/2/e00191-16">Deblur</a>, <a href="https://www.nature.com/articles/nmeth.3869">DADA2</a>, <a href="https://www.frontiersin.org/articles/10.3389/fgene.2019.01366/full">NG-Tax</a>, etc. You will get a table of taxa abundances and taxonomic classification of taxa. This can be merged with metadata and used for downstream analysis.</p>
<p>Data can be in <a href="http://biom-format.org/">*.biom</a> format or tables.</p>
<p>Let’s get started!<br />
Importing data into R can be the first major hurdle. The most common mistake that I have noticed is issue with file location. RStudio in combination with R is awesome for “containerizing” data analysis. By creating an RStudio project, it is possible to set path within the project folder. Briefly, this is demonstrated in documentation that we created previously <a href="https://mibwurrepo.github.io/Microbial-bioinformatics-introductory-course-Material-2018/introduction.html">here</a>. There, you can find a guide to do several common analysis with support of available R packages.</p>
<p>Here, I will focus on basics of handling microbiome data using phyloseq and microbiome.</p>
<p>Check the <code>read_phyloseq</code> function from microbiome package for importing and converting you data into phyloseq format.</p>
<p>Assuming (will try to avoid this as much as possible) that you have managed to import your data in R, we will see how to do basic data operations. This will be an introduction to helper functions in microbiome and phyloseq packages. Unfortunately, many users do not go through the <a href="https://microbiome.github.io/tutorials/Preprocessing.html">documentation</a> and I hope that you don’t do the same with this page!</p>
<p>First install the packages of interest.</p>
<pre class="r"><code># From cran  
install.packages(&quot;devtools&quot;)

# From Bioconductor
if (!requireNamespace(&quot;BiocManager&quot;, quietly = TRUE)) # Install BiocManager if not already installed. 
    install.packages(&quot;BiocManager&quot;)

BiocManager::install(&quot;microbiome&quot;)

# From GitHub
devtools::install_github(&quot;microsud/microbiomeutilities&quot;)</code></pre>
<p>Note: When possible I will try to call the functions together with package name for the purpose of clarity of packages providing the function. E.g. <code>microbiome::transform()</code></p>
<div id="load-r-package" class="section level2">
<h2>Load R package</h2>
<pre class="r"><code># Load a library  
library(microbiomeutilities)</code></pre>
</div>
<div id="load-example-data" class="section level2">
<h2>Load example data</h2>
<p>We use the data stored in <code>microbiomeutilties</code> R package.</p>
<pre class="r"><code>data(&quot;zackular2014&quot;) # access the test data
# assign to an aboject with shorter name.
ps &lt;- zackular2014</code></pre>
</div>
<div id="get-to-know-your-data" class="section level2">
<h2>Get to know your data</h2>
<div id="check" class="section level3">
<h3>Check</h3>
<p>Firstly, check and understand some basic information about your data.</p>
<pre class="r"><code>microbiomeutilities::print_ps(ps)</code></pre>
<pre><code>## 01] ntaxa = 2078</code></pre>
<pre><code>## 02] nsamples = 88</code></pre>
<pre><code>## 03] nsamplesvariables = 7</code></pre>
<pre><code>## 04] nranks = 7</code></pre>
<pre><code>## 05] Min. number of reads = 15704</code></pre>
<pre><code>## 06] Max. number of reads = 216888</code></pre>
<pre><code>## 07] Total number of reads = 4594626</code></pre>
<pre><code>## 08] Average number of reads = 52211.66</code></pre>
<pre><code>## 09] Median number of reads = 49415.5</code></pre>
<pre><code>## 10] Sparsity = 0.728426590252865</code></pre>
<pre><code>## 11] Number of singletons = 0</code></pre>
<pre><code>## 12] % of taxa that are singletons 
##   (i.e. exactly one read detected across all samples) = 0</code></pre>
<p><strong>Questions you need to ask regarding your data:</strong><br />
How many taxa are present in the data <code>ntaxa</code> ?<br />
Do you expect these number of taxa in your ecosystem of interest?<br />
Have you successfully imported all your samples <code>nsamples</code>?<br />
Are all the taxonomic ranks included in your taxonomy <code>nranks</code>?<br />
What is the Minimum and Maximum number of reads in your samples?<br />
Are there singletons?<br />
Is your data zero-inflated <code>Sparsity</code>?<br />
If there are high number of zeroes, what does it mean for your analysis approach?<br />
<a href="https://www.google.com/">Google</a> is your friend (but be careful).</p>
</div>
<div id="access-parts" class="section level3">
<h3>Access parts</h3>
<pre class="r"><code># OTU table  

otu_tab &lt;- microbiome::abundances(ps)

# check 

otu_tab[1:5,1:5] # for my table show me [1 to 5 rows, 1 to 5 columns]</code></pre>
<pre><code>##               Adenoma10-2757 Adenoma11-2775 Adenoma13-2803 Adenoma14-2817
## d__denovo1773              0              0              0              0
## d__denovo1771              0              0              0              6
## d__denovo1776              0              0              3              0
## d__denovo1777              0              0              0             15
## d__denovo1775              1              0              1              1
##               Adenoma15-2847
## d__denovo1773              0
## d__denovo1771              0
## d__denovo1776              0
## d__denovo1777              1
## d__denovo1775              4</code></pre>
<pre class="r"><code># Taxonomy table
tax_tab &lt;- phyloseq::tax_table(ps)
# check 
tax_tab[1:5,1:5] # for my table show me [1 to 5 otu ids, 1 to 5 first five ranks]</code></pre>
<pre><code>## Taxonomy Table:     [5 taxa by 5 taxonomic ranks]:
##               Domain        Phylum             Class           
## d__denovo1773 &quot;k__Bacteria&quot; &quot;p__Bacteroidetes&quot; &quot;c__Bacteroidia&quot;
## d__denovo1771 &quot;k__Bacteria&quot; &quot;p__Bacteroidetes&quot; &quot;c__Bacteroidia&quot;
## d__denovo1776 &quot;k__Bacteria&quot; &quot;p__Firmicutes&quot;    &quot;c__Clostridia&quot; 
## d__denovo1777 &quot;k__Bacteria&quot; &quot;p__Bacteroidetes&quot; &quot;c__Bacteroidia&quot;
## d__denovo1775 &quot;k__Bacteria&quot; &quot;p__Firmicutes&quot;    &quot;c__Clostridia&quot; 
##               Order              Family              
## d__denovo1773 &quot;o__Bacteroidales&quot; &quot;f__Bacteroidaceae&quot; 
## d__denovo1771 &quot;o__Bacteroidales&quot; &quot;f__Bacteroidaceae&quot; 
## d__denovo1776 &quot;o__Clostridiales&quot; &quot;f__Ruminococcaceae&quot;
## d__denovo1777 &quot;o__Bacteroidales&quot; &quot;f__Bacteroidaceae&quot; 
## d__denovo1775 &quot;o__Clostridiales&quot; &quot;f__Lachnospiraceae&quot;</code></pre>
<p>Notice something, very common to see patterns like "k__" prefixes and OTU id "d__denovo" or sequences as row names.</p>
</div>
</div>
<div id="clean-taxonomy-table" class="section level2">
<h2>Clean taxonomy table</h2>
<p>Cleaning of taxonomy tables is useful to do at the beginning of the analysis. It is time-consuming but also useful to understanding taxonomic information of your taxa.</p>
<p>Let us use two functions to change OTU ids. <code>gsub</code> from base R and <code>taxa_names</code> from phyloseq.</p>
<pre class="r"><code># accessing the OTUids 
taxa_names(ps)[1:5] # print first 5 ids</code></pre>
<pre><code>## [1] &quot;d__denovo1773&quot; &quot;d__denovo1771&quot; &quot;d__denovo1776&quot; &quot;d__denovo1777&quot;
## [5] &quot;d__denovo1775&quot;</code></pre>
<pre class="r"><code># find and substitute
taxa_names(ps) &lt;- gsub(taxa_names(ps), pattern = &quot;d__&quot;, replacement = &quot;&quot;)  

# Check again Taxonomy table
phyloseq::tax_table(ps)[1:3,1:3] # check for my table show me [1 to 3 otu ids, 1 to 3 first three ranks]. </code></pre>
<pre><code>## Taxonomy Table:     [3 taxa by 3 taxonomic ranks]:
##            Domain        Phylum             Class           
## denovo1773 &quot;k__Bacteria&quot; &quot;p__Bacteroidetes&quot; &quot;c__Bacteroidia&quot;
## denovo1771 &quot;k__Bacteria&quot; &quot;p__Bacteroidetes&quot; &quot;c__Bacteroidia&quot;
## denovo1776 &quot;k__Bacteria&quot; &quot;p__Firmicutes&quot;    &quot;c__Clostridia&quot;</code></pre>
<p>What happened here? We first accessed the names of taxa (OTUids) with <code>taxa_names(ps)</code> which contains the names of OTUids.</p>
<pre class="r"><code>taxa_names(ps)[1:5] # print first 5 names</code></pre>
<pre><code>## [1] &quot;denovo1773&quot; &quot;denovo1771&quot; &quot;denovo1776&quot; &quot;denovo1777&quot; &quot;denovo1775&quot;</code></pre>
<p>For each of these, we ask <code>gsub</code> to find a pattern "d__" and substitute it with "" (nothing).<br />
By providing “taxa_names(ps) &lt;-” we straight away substitute these names.</p>
<p>We can substitute “denovo” with just “OTU-”</p>
<pre class="r"><code># find and substitute
taxa_names(ps) &lt;- gsub(taxa_names(ps), pattern = &quot;denovo&quot;, replacement = &quot;OTU-&quot;)  </code></pre>
<p>Next, cleaning the "k__" and similar values.</p>
<pre class="r"><code># extending gsub to entire table 
tax_table(ps)[, colnames(tax_table(ps))] &lt;- gsub(tax_table(ps)[, colnames(tax_table(ps))],     pattern = &quot;[a-z]__&quot;, replacement = &quot;&quot;)</code></pre>
<p>What happened here? By using <code>colnames(tax_table(ps))</code> we are specifying names of the columns in <code>tax_table(ps)</code>.</p>
<pre class="r"><code>colnames(tax_table(ps))</code></pre>
<pre><code>## [1] &quot;Domain&quot;  &quot;Phylum&quot;  &quot;Class&quot;   &quot;Order&quot;   &quot;Family&quot;  &quot;Genus&quot;   &quot;Species&quot;</code></pre>
<p>For columns in <code>tax_table(ps)</code> we used <code>gsub</code> to check all the columns for patterns ranging from <code>[a-z]</code> joined by <code>__</code> like this <code>[a-z]__</code> and substitute it with "" i.e. nothing.</p>
<pre class="r"><code>phyloseq::tax_table(ps)[1:3,1:3] # check for my table show me [1 to 3 otu ids, 1 to 3 first three ranks].</code></pre>
<pre><code>## Taxonomy Table:     [3 taxa by 3 taxonomic ranks]:
##          Domain     Phylum          Class        
## OTU-1773 &quot;Bacteria&quot; &quot;Bacteroidetes&quot; &quot;Bacteroidia&quot;
## OTU-1771 &quot;Bacteria&quot; &quot;Bacteroidetes&quot; &quot;Bacteroidia&quot;
## OTU-1776 &quot;Bacteria&quot; &quot;Firmicutes&quot;    &quot;Clostridia&quot;</code></pre>
<p>There are several other special characters and labels that you should check for in your data. This is an important step, especially when aggregating/collapsing at higher taxonomic ranks.</p>
<p>Some examples are below. These are not detected in the test data used here. However, I have come across these in my research.</p>
<pre class="r"><code># Replace sequence IDs with ATACAACTATACG with ASV1 and so on...
taxa_names(ps) &lt;- paste0(&quot;ASV&quot;, seq(ntaxa(ps)))</code></pre>
<p>Replace special characters.</p>
<pre class="r"><code>tax_table(ps)[,colnames(tax_table(ps))] &lt;- gsub(tax_table(ps)[,colnames(tax_table(ps))],pattern=&quot;=*&quot;,replacement=&quot;&quot;)

# remove asterisk
tax_table(ps)[,colnames(tax_table(ps))] &lt;- gsub(tax_table(ps)[,colnames(tax_table(ps))],pattern=&quot;[*]&quot;,replacement=&quot;&quot;) # notice here the square brackets. These are required when special characters such as * and ~ below are used as they have functional role in R base programming. 

# remove ~
tax_table(ps)[,colnames(tax_table(ps))] &lt;- gsub(tax_table(ps)[,colnames(tax_table(ps))],pattern=&quot;[~]&quot;,replacement=&quot;&quot;)</code></pre>
<p>Other than <code>gsub</code>, we can also go through one column at a time in the taxonomy table to specifically substitute ambiguous markings.</p>
<pre class="r"><code># For each column separately we attempt to replace &quot;k__&lt;empty&gt;&quot; with &quot;&quot; for consistency.
tax_table(ps)[tax_table(ps) == &quot;k__&lt;empty&gt;&quot;] &lt;- &quot;&quot;
tax_table(ps)[tax_table(ps) == &quot;p__&lt;empty&gt;&quot;] &lt;- &quot;&quot;
tax_table(ps)[tax_table(ps) == &quot;c__&lt;empty&gt;&quot;] &lt;- &quot;&quot;
tax_table(ps)[tax_table(ps) == &quot;o__&lt;empty&gt;&quot;] &lt;- &quot;&quot;
tax_table(ps)[tax_table(ps) == &quot;f__&lt;empty&gt;&quot;] &lt;- &quot;&quot;
tax_table(ps)[tax_table(ps) == &quot;g__&lt;empty&gt;&quot;] &lt;- &quot;&quot;
# some more ambiguities 
tax_table(ps)[tax_table(ps) == &quot;o__Unknown_Order&quot;] &lt;- &quot;&quot;
tax_table(ps)[tax_table(ps) == &quot;c__Unknown_Class&quot;] &lt;- &quot;&quot;
tax_table(ps)[tax_table(ps) == &quot;f__Unknown_Family&quot;] &lt;- &quot;&quot;</code></pre>
<p>What happened here? We asked for any pattern matching "k__<empty>" in the table and replace it with "" nothing.</p>
<p>Aggregate at genus level.</p>
<pre class="r"><code>ps.gen &lt;- phyloseq::tax_glom(ps, &quot;Genus&quot;, NArm = TRUE)</code></pre>
<p>Notice the <code>NArm = TRUE</code> parameter. This will remove anything that is unclassified at genus level. This choice will depend on your research goals.<br />
Check the names.</p>
<pre class="r"><code>taxa_names(ps.gen)[1:5]</code></pre>
<pre><code>## [1] &quot;OTU-174&quot;  &quot;OTU-1079&quot; &quot;OTU-363&quot;  &quot;OTU-369&quot;  &quot;OTU-103&quot;</code></pre>
<p>Substitute these IDs with names of genus.</p>
<pre class="r"><code>taxa_names(ps.gen) &lt;- tax_table(ps.gen)[,&quot;Genus&quot;]</code></pre>
<p>Check if the <code>gsub</code> above worked.</p>
<pre class="r"><code>taxa_names(ps.gen)[1:5]</code></pre>
<pre><code>## [1] &quot;Listeria&quot;         &quot;Allisonella&quot;      &quot;Enterorhabdus&quot;    &quot;Cloacibacillus&quot;  
## [5] &quot;Clostridium_XlVb&quot;</code></pre>
<p>What happened here? Similar to our first instance of <code>gsub</code> operation to replace "d__", we asked R to substitute <code>taxa_names(ps.gen)</code> which are ids with corresponding genus identity accessed by this <code>tax_table(ps.gen)[,"Genus"]</code></p>
<pre class="r"><code>tax_table(ps.gen)[,&quot;Genus&quot;][1:5] # [1:5] is first five to print. </code></pre>
<pre><code>## Taxonomy Table:     [5 taxa by 1 taxonomic ranks]:
##                  Genus             
## Listeria         &quot;Listeria&quot;        
## Allisonella      &quot;Allisonella&quot;     
## Enterorhabdus    &quot;Enterorhabdus&quot;   
## Cloacibacillus   &quot;Cloacibacillus&quot;  
## Clostridium_XlVb &quot;Clostridium_XlVb&quot;</code></pre>
<p>Print all genus names. We use the unique() function to print each name only once.</p>
<pre class="r"><code># 
unique(tax_table(ps.gen)[,&quot;Genus&quot;] )</code></pre>
<pre><code>## Taxonomy Table:     [104 taxa by 1 taxonomic ranks]:
##                                    Genus                               
## Listeria                           &quot;Listeria&quot;                          
## Allisonella                        &quot;Allisonella&quot;                       
## Enterorhabdus                      &quot;Enterorhabdus&quot;                     
## Cloacibacillus                     &quot;Cloacibacillus&quot;                    
## Clostridium_XlVb                   &quot;Clostridium_XlVb&quot;                  
## Clostridium_XlVa                   &quot;Clostridium_XlVa&quot;                  
## Subdoligranulum                    &quot;Subdoligranulum&quot;                   
## Streptococcus                      &quot;Streptococcus&quot;                     
## Anaerococcus                       &quot;Anaerococcus&quot;                      
## Gemella                            &quot;Gemella&quot;                           
## Sporobacter                        &quot;Sporobacter&quot;                       
## Acinetobacter                      &quot;Acinetobacter&quot;                     
## Streptophyta                       &quot;Streptophyta&quot;                      
## Slackia                            &quot;Slackia&quot;                           
## Acidaminococcus                    &quot;Acidaminococcus&quot;                   
## Citrobacter                        &quot;Citrobacter&quot;                       
## Desulfovibrio                      &quot;Desulfovibrio&quot;                     
## Fusobacterium                      &quot;Fusobacterium&quot;                     
## Prevotella                         &quot;Prevotella&quot;                        
## Actinomyces                        &quot;Actinomyces&quot;                       
## Clostridium_XI                     &quot;Clostridium_XI&quot;                    
## Gemmiger                           &quot;Gemmiger&quot;                          
## Finegoldia                         &quot;Finegoldia&quot;                        
## Insolitispirillum                  &quot;Insolitispirillum&quot;                 
## Solobacterium                      &quot;Solobacterium&quot;                     
## Gordonibacter                      &quot;Gordonibacter&quot;                     
## Lactonifactor                      &quot;Lactonifactor&quot;                     
## Oscillibacter                      &quot;Oscillibacter&quot;                     
## Acetanaerobacterium                &quot;Acetanaerobacterium&quot;               
## Staphylococcus                     &quot;Staphylococcus&quot;                    
## Rothia                             &quot;Rothia&quot;                            
## Enterobacter                       &quot;Enterobacter&quot;                      
## Veillonella                        &quot;Veillonella&quot;                       
## Coprobacillus                      &quot;Coprobacillus&quot;                     
## Bilophila                          &quot;Bilophila&quot;                         
## Megasphaera                        &quot;Megasphaera&quot;                       
## Anaerosporobacter                  &quot;Anaerosporobacter&quot;                 
## Lactococcus                        &quot;Lactococcus&quot;                       
## Akkermansia                        &quot;Akkermansia&quot;                       
## Methanobrevibacter                 &quot;Methanobrevibacter&quot;                
## Collinsella                        &quot;Collinsella&quot;                       
## Dialister                          &quot;Dialister&quot;                         
## Ruminococcus2                      &quot;Ruminococcus2&quot;                     
## Dorea                              &quot;Dorea&quot;                             
## Marvinbryantia                     &quot;Marvinbryantia&quot;                    
## Blautia                            &quot;Blautia&quot;                           
## Bacteroides                        &quot;Bacteroides&quot;                       
## Escherichia/Shigella               &quot;Escherichia/Shigella&quot;              
## Roseburia                          &quot;Roseburia&quot;                         
## Anaerostipes                       &quot;Anaerostipes&quot;                      
## Holdemania                         &quot;Holdemania&quot;                        
## Parabacteroides                    &quot;Parabacteroides&quot;                   
## Faecalibacterium                   &quot;Faecalibacterium&quot;                  
## Clostridium_IV                     &quot;Clostridium_IV&quot;                    
## Eubacterium                        &quot;Eubacterium&quot;                       
## Barnesiella                        &quot;Barnesiella&quot;                       
## Deinococcus                        &quot;Deinococcus&quot;                       
## Neisseria                          &quot;Neisseria&quot;                         
## Hydrogenoanaerobacterium           &quot;Hydrogenoanaerobacterium&quot;          
## Odoribacter                        &quot;Odoribacter&quot;                       
## Enterococcus                       &quot;Enterococcus&quot;                      
## Anaerotruncus                      &quot;Anaerotruncus&quot;                     
## Parasutterella                     &quot;Parasutterella&quot;                    
## Salmonella                         &quot;Salmonella&quot;                        
## Butyricimonas                      &quot;Butyricimonas&quot;                     
## Peptoniphilus                      &quot;Peptoniphilus&quot;                     
## Lachnospiracea_incertae_sedis      &quot;Lachnospiracea_incertae_sedis&quot;     
## Pseudomonas                        &quot;Pseudomonas&quot;                       
## Coprococcus                        &quot;Coprococcus&quot;                       
## Leuconostoc                        &quot;Leuconostoc&quot;                       
## Helicobacter                       &quot;Helicobacter&quot;                      
## Catenibacterium                    &quot;Catenibacterium&quot;                   
## Anaerofustis                       &quot;Anaerofustis&quot;                      
## Eggerthella                        &quot;Eggerthella&quot;                       
## Anaerovorax                        &quot;Anaerovorax&quot;                       
## Parvimonas                         &quot;Parvimonas&quot;                        
## Hallella                           &quot;Hallella&quot;                          
## Anaerofilum                        &quot;Anaerofilum&quot;                       
## Mannheimia                         &quot;Mannheimia&quot;                        
## Turicibacter                       &quot;Turicibacter&quot;                      
## Lactobacillus                      &quot;Lactobacillus&quot;                     
## Butyricicoccus                     &quot;Butyricicoccus&quot;                    
## Clostridium_XVIII                  &quot;Clostridium_XVIII&quot;                 
## Herbaspirillum                     &quot;Herbaspirillum&quot;                    
## Pseudoflavonifractor               &quot;Pseudoflavonifractor&quot;              
## Paraprevotella                     &quot;Paraprevotella&quot;                    
## Ruminococcus                       &quot;Ruminococcus&quot;                      
## Alistipes                          &quot;Alistipes&quot;                         
## Bifidobacterium                    &quot;Bifidobacterium&quot;                   
## Sutterella                         &quot;Sutterella&quot;                        
## Howardella                         &quot;Howardella&quot;                        
## Flavonifractor                     &quot;Flavonifractor&quot;                    
## Mogibacterium                      &quot;Mogibacterium&quot;                     
## Erysipelotrichaceae_incertae_sedis &quot;Erysipelotrichaceae_incertae_sedis&quot;
## Rhodobacter                        &quot;Rhodobacter&quot;                       
## Weissella                          &quot;Weissella&quot;                         
## Oxalobacter                        &quot;Oxalobacter&quot;                       
## Clostridium_III                    &quot;Clostridium_III&quot;                   
## Haemophilus                        &quot;Haemophilus&quot;                       
## Clostridium_sensu_stricto          &quot;Clostridium_sensu_stricto&quot;         
## Porphyromonas                      &quot;Porphyromonas&quot;                     
## Phascolarctobacterium              &quot;Phascolarctobacterium&quot;             
## Atopobium                          &quot;Atopobium&quot;                         
## Hespellia                          &quot;Hespellia&quot;</code></pre>
<p>There can also be spaces between names e.g <code>Prevotella 1</code> which can be changed to <code>Prevotella_1</code></p>
<pre class="r"><code>taxa_names(ps) &lt;- gsub(&quot; &quot;, &quot;.&quot;, taxa_names(ps))</code></pre>
<p>There can also be spaces between names e.g <code>[Prevotella] copri</code> which can be changed to <code>Prevotella copri</code></p>
<pre class="r"><code>taxa_names(ps) &lt;- gsub(&quot;\\[|\\]&quot;, &quot;&quot;, taxa_names(ps))</code></pre>
<p>There can also be spaces between names e.g <code>Prevotella-copri</code> which can be changed to <code>Prevotella.copri</code></p>
<pre class="r"><code>taxa_names(ps) &lt;- gsub(&quot;-&quot;, &quot;.&quot;, taxa_names(ps))</code></pre>
<p>Also check the <code>format_to_besthit</code> in <code>microbiomeutilities</code> R package <a href="https://microsud.github.io/microbiomeutilities/reference/format_to_besthit.html">here</a>.</p>
<p>These are just the common examples I observed in my research. There can be may other challenges with taxonomic labeling. However, I hope this has given a general impression of how we can do cleaning of data in R with just <code>gsub</code> and <code>phyloseq</code> functions to access information from within a <code>phyloseq</code> object.<br />
For more information of <code>gsub</code> and related <code>grepl</code> check the base R <a href="https://stat.ethz.ch/R-manual/R-devel/library/base/html/grep.html">Pattern Matching and Replacement</a></p>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
