<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Select Peer-Reviewed Publications</title>

<script src="site_libs/header-attrs-2.8/header-attrs.js"></script>
<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>





<link rel="stylesheet" href="portfolio.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Sudarshan Shetty</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="Research_Projects.html">Research</a>
</li>
<li>
  <a href="Software.html">Software and codes</a>
</li>
<li>
  <a href="Publications.html">Publications</a>
</li>
<li>
  <a href="teaching.html">Teaching</a>
</li>
<li>
  <a href="Science_communication.html">Science communication</a>
</li>
<li>
  <a href="Other_Informations.html">Other Information</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Select Peer-Reviewed Publications</h1>

</div>


<p><img src="images/icons/open-access.svg" class = "open-access"/> Open access</p>
<blockquote>
<p><strong>2021</strong></p>
</blockquote>
<p>Morales D, <strong>Shetty, SA.</strong>, López-Plaza B, Gómez-Candela C, Smidt H, Marín FR, Soler-Rivas C. Modulation of human intestinal microbiota in a clinical trial by consumption of a β-d-glucan-enriched extract obtained from Lentinula edodes. 2021 Feb 12:1-7; <a href="https://link.springer.com/article/10.1007/s00394-021-02504-4">European Journal of Nutrition</a></p>
<blockquote>
<p><strong>2020</strong></p>
</blockquote>
<p><strong>Shetty, SA.</strong>, Boeren S, Bui TPN, Smidt H, and De Vos WM, 2020. Unravelling lactate‐acetate and sugar conversion into butyrate by intestinal <em>Anaerobutyricum</em> and <em>Anaerostipes</em> species by comparative proteogenomics. 2020 October; <a href="https://sfamjournals.onlinelibrary.wiley.com/doi/pdf/10.1111/1462-2920.15269">Environmental Microbiology</a> <img src="images/icons/open-access.svg" class = "open-access"/></p>
<p>Van der Graaf CM, Sánchez-España J, Yusta I, Ilin A, <strong>Shetty SA</strong>, Bale NJ, Villanueva L, Stams AJ, Sánchez-Andrea I. Biosulfidogenesis mediates natural attenuation in acidic mine pit lakes. Microorganisms. 2020 Sep;8(9):1275. <img src="images/icons/open-access.svg" class = "open-access"/></p>
<p>Jacksch S, Thota J, <strong>Shetty S</strong>, Smidt H, Schnell S, Egert M. Metagenomic Analysis of Regularly Microwave-Treated and Untreated Domestic Kitchen Sponges. Microorganisms. 2020 May;8(5):736. <img src="images/icons/open-access.svg" class = "open-access"/></p>
<p>Peng P, Lu Y, Bosma TN, Nijenhuis I, Nijsse B, <strong>Shetty SA</strong>, Ruecker A, Umanets A, Ramiro-Garcia J, Kappler A, Sipkema D. Metagenomic-and cultivation-based exploration of anaerobic chloroform biotransformation in hypersaline sediments as natural source of chloromethanes. Microorganisms. 2020 May;8(5):665. <img src="images/icons/open-access.svg" class = "open-access"/></p>
<p>Edwards JE, <strong>Shetty SA</strong>, van den Berg P, Burden F, van Doorn DA, Pellikaan WF, Dijkstra J, Smidt H. Multi-kingdom characterization of the core equine fecal microbiota based on multiple equine (sub)species. Animal Microbiome 2020 2:6 <img src="images/icons/open-access.svg" class = "open-access"/></p>
<p>Gutleben J, Loureiro C, Ramírez Romero LA, <strong>Shetty S</strong>, Wijffels RH, Smidt H, Sipkema D. Cultivation of Bacteria From Aplysina aerophoba: Effects of Oxygen and Nutrient Gradients. Frontiers in Microbiology. 2020 Feb 19;11:175. <img src="images/icons/open-access.svg" class = "open-access"/></p>
<blockquote>
<p><strong>2019</strong></p>
</blockquote>
<p><strong>Shetty SA</strong>, Lahti L. Microbiome data science. Journal of biosciences. 2019 Dec 1;44(5):115. <img src="images/icons/open-access.svg" class = "open-access"/></p>
<p>Hynönen U, Zoetendal EG, Virtala AM, <strong>Shetty S</strong>, Hasan S, Jakava-Viljanen M, de Vos WM, Palva A. Molecular ecology of the yet uncultured bacterial Ct85-cluster in the mammalian gut. Anaerobe. 2019 Sep 25:102104.</p>
<p><strong>Shetty SA</strong>, Smidt H, de Vos WM. Reconstructing functional networks in the human intestinal tract using synthetic microbiomes. Current Opinion in Biotechnology. 2019 Aug 1;58:146-54. <img src="images/icons/open-access.svg" class = "open-access"/></p>
<p>Bodkhe R, <strong>Shetty SA</strong>, Dhotre DP, Verma AK, Bhatia K, Mishra A, Kaur G, Pande P, Bangarusamy DK, Santosh BP, Perumal RC, Ahuja V, Makharia GK, Shouche YS. Comparison of Small Gut and Whole Gut Microbiota of First-Degree Relatives with Adult Patients with Celiac Disease and Controls. Front Microbiol. 2019. doi: 10.3389/fmicb.2019.00164. <img src="images/icons/open-access.svg" class = "open-access"/></p>
<blockquote>
<p><strong>2018</strong></p>
</blockquote>
<p><strong>Shetty SA</strong>, Zuffa S, Bui TP, Aalvink S, Smidt H, De Vos WM. Reclassification of Eubacterium hallii as Anaerobutyricum hallii gen. nov., comb. nov., and description of Anaerobutyricum soehngenii sp. nov., a butyrate and propionate-producing bacterium from infant faeces. International Journal of Systematic and Evolutionary Microbiology. 2018 Oct 23. <img src="images/icons/open-access.svg" class = "open-access"/></p>
<p>Atashgahi S, <strong>Shetty SA</strong>, Smidt H, De Vos WM. Flux, impact and fate of halogenated xenobiotic compounds in the gut. Frontiers in physiology. 2018; 9:888. <img src="images/icons/open-access.svg" class = "open-access"/></p>
<p>Ghattargi VC, Nimonkar YS, Burse SA, Davray D, Kumbhare SV, <strong>Shetty SA</strong>, Gaikwad MA, Suryavanshi MV, Doijad SP, Utage B, Sharma OP. Genomic and physiological analyses of an indigenous strain, <em>Enterococcus faecium</em> 17OM39. Functional &amp; integrative genomics. 2018 Jul 1;18(4):385-99.</p>
<blockquote>
<p><strong>2017</strong></p>
</blockquote>
<p><strong>Shetty SA</strong>, Ritari J, Paulin L, Smidt H, De Vos WM. Complete Genome Sequence of <em>Eubacterium hallii</em> Strain L2-7. Genome Announc. 2017 Oct 26;5(43). pii: e01167-17. <img src="images/icons/open-access.svg" class = "open-access"/></p>
<p><strong>Shetty SA</strong>#, Hugenholtz F, Lahti L, Smidt H, de Vos WM. Intestinal microbiome landscaping: insight in community assemblage and implications for microbial modulation strategies. FEMS Microbiol Rev. 2017 Mar 1;41(2):182-199. <img src="images/icons/open-access.svg" class = "open-access"/></p>
<p>Buntin N, Hongpattarakere T, Ritari J, Douillard FP, Paulin L, Boeren S, <strong>Shetty SA</strong>, de Vos WM. An inducible operon is involved in inulin utilization in <em>Lactobacillus plantarum</em> strains, as revealed by comparative proteogenomics and metabolic profiling. Applied and Environmental Microbiology. 2017 Jan 15;83(2):e02402-16.</p>
<blockquote>
<p><strong>2016</strong></p>
</blockquote>
<p>Marathe NP, <strong>Shetty SA</strong>, Shouche YS, Larsson DG. Limited Bacterial Diversity within a Treatment Plant Receiving Antibiotic-Containing Waste from Bulk Drug Production. PLoS One. 2016 Nov 3;11(11):e0165914. <img src="images/icons/open-access.svg" class = "open-access"/></p>
<p>Bui TP, <strong>Shetty SA</strong>, Lagkouvardos I, Ritari J, Chamlagain B, Douillard FP, Paulin L, Piironen V, Clavel T, Plugge CM, de Vos WM. Comparative genomics and physiology of the butyrate-producing bacterium <em>Intestinimonas butyriciproducens</em>. Environ Microbiol Rep. 2016 Dec;8(6):1024-1037. <img src="images/icons/open-access.svg" class = "open-access"/></p>
<p>Paul D, Kumbhare SV, Mhatre SS, Chowdhury SP, <strong>Shetty SA</strong>, Marathe NP, Bhute S, Shouche YS. Exploration of Microbial Diversity and Community Structure of Lonar Lake: The Only Hypersaline Meteorite Crater Lake within Basalt Rock. Front Microbiol. 2016 Jan 22; 6:1553. <img src="images/icons/open-access.svg" class = "open-access"/></p>
<p>Bhute S, Pande P, <strong>Shetty SA</strong>, Shelar R, Mane S, Kumbhare SV, Gawali A, Makhani H, Navandar M, Dhotre D, Lubree H, Agarwal D, Patil R, Ozarkar S, Ghaskadbi S, Yajnik C, Juvekar S, Makharia GK, Shouche YS. Molecular Characterization and Meta-Analysis of Gut Microbial Communities Illustrate Enrichment of <em>Prevotella</em> and <em>Megasphaera</em> in Indian Subjects. Front Microbiol. 2016 May 9; 7:660. <img src="images/icons/open-access.svg" class = "open-access"/></p>
<blockquote>
<p><strong>2015</strong></p>
</blockquote>
<p>Pandey PK, Verma P, Marathe N, <strong>Shetty SA</strong>, Bavdekar A, Patole MS, Stensvold CR, Shouche YS. Prevalence and subtype analysis of <em>Blastocystis</em> in healthy Indian individuals. Infect Genet Evol. 2015 Apr; 31:296-9.</p>
<blockquote>
<p><strong>2014</strong></p>
</blockquote>
<p>Marathe NP, <strong>Shetty SA</strong>, Lanjekar VB, Rasane MH, Ranade DR, Shouche YS. Genome sequencing of multidrug resistant novel <em>Clostridium</em> sp. BL8 reveals its potential for pathogenicity. Gut Pathog. 2014 Jul 18; 6:30. <img src="images/icons/open-access.svg" class = "open-access"/></p>
<p>Mahale KN, Paranjape PS, Marathe NP, Dhotre DP, Chowdhury S, <strong>Shetty SA</strong>, Sharma A, Sharma K, Tuteja U, Batra HV, Shouche YS. Draft Genome Sequences of <em>Yersinia pestis</em> Strains from the 1994 Plague Epidemic of Surat and 2002 Shimla Outbreak in India. Indian J Microbiol. 2014 Dec;54(4):480-2. <img src="images/icons/open-access.svg" class = "open-access"/></p>
<p>Kulkarni GJ, <strong>Shetty SA</strong>, Dharne MS, Shouche YS. Genome sequencing analysis reveals virulence-related gene content of <em>Ochrobactrum intermedium</em> strain 229E, a urease-positive strain isolated from the human gastric niche. FEMS Microbiol Lett. 2014 Oct;359(1):12-5. <img src="images/icons/open-access.svg" class = "open-access"/></p>
<blockquote>
<p><strong>2013</strong></p>
</blockquote>
<p><strong>Shetty SA</strong>, Marathe NP, Shouche YS. Opportunities and challenges for gut microbiome studies in the Indian population. Microbiome. 2013 Sep 17;1(1):24. <img src="images/icons/open-access.svg" class = "open-access"/></p>
<p><strong>Shetty SA</strong>, Marathe NP, Lanjekar V, Ranade D, Shouche YS. Comparative genome analysis of <em>Megasphaera</em> sp. reveals niche specialization and its potential role in the human gut. PLoS One. 2013 Nov 18; 8(11):e79353. <img src="images/icons/open-access.svg" class = "open-access"/></p>
<p>Kulkarni G, Dhotre D, Dharne M, <strong>Shetty SA</strong>, Chowdhury S, Misra V, Misra S, Patole M, Shouche Y. Draft genome of <em>Ochrobactrum intermedium</em> strain M86 isolated from non-ulcer dyspeptic individual from India. Gut Pathog. 2013 Apr 4; 5(1):7. <img src="images/icons/open-access.svg" class = "open-access"/></p>
<blockquote>
<p><strong>2012</strong></p>
</blockquote>
<p>Marathe N, <strong>Shetty SA</strong>, Lanjekar V, Ranade D, Shouche Y. Changes in human gut flora with age: an Indian familial study. BMC Microbiol. 2012 Sep 26; 12:222. <img src="images/icons/open-access.svg" class = "open-access"/></p>
<p><strong>Shetty SA</strong>, Dhotre DP, Shetty SA, Chowdhury SP, Chaudhari BL, Shouche YS. Genome sequence of <em>Janibacter hoylei</em> MTCC8307, isolated from the stratospheric air. J Bacteriol. 2012 Dec;194(23):6629-30. <img src="images/icons/open-access.svg" class = "open-access"/></p>
<blockquote>
<p><strong>Manuscripts in pre-print:</strong></p>
</blockquote>
<p><strong>Shetty SA</strong>#. Gut microbiota features of the geographically diverse Indian population. <a href="https://www.biorxiv.org/content/10.1101/478586v1.full">bioRxiv</a> (2018). <img src="images/icons/open-access.svg" class = "open-access"/></p>
<blockquote>
<p><strong>Academic Profile </strong></p>
</blockquote>
<span id="badgeCont856">
<script type="text/javascript" src="https://publons.com/mashlets?el=badgeCont856&rid=L-2278-2019"></script>
<p></span></p>
<hr />
<div class="column-left">

</div>
<p>
Copyright © 2021 Sudarshan Shetty. All rights reserved.
</p>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
